platform/MCU/NRF51/SPARK_Firmware_Driver/src/data_services.cpp

```diff
@@ -18,6 +18,7 @@
 #include "data_management_layer.h"
 #include "info_data_service.h"
 #include "socket_manager.h"
+#include "custom_data_service.h"
```

Comment:
don't know why this is here in this commit, either. The pull request was SUPPOSED to only have changes since BluzDK ... develop HEAD. :-/